components/builder-depot/src/handlers/integrations.rs

```diff
@@ -74,6 +71,32 @@ pub fn validate_params(
     Ok(res)
 }
 
+pub fn fetch_origin_integrations(req: &mut Request) -> IronResult<Response> {
+    let params = match validate_params(req, &["origin"]) {
+        Ok(p) => p,
+        Err(st) => return Ok(Response::with(st)),
+    };
+    let mut request = OriginIntegrationRequest::new();
+    request.set_origin(params["origin"].clone());
+    match route_message::<OriginIntegrationRequest, OriginIntegrationResponse>(req, &request) {
+        Ok(oir) => {
+            println!("{:?}", &oir.get_integrations());
```

Comment:
It'd be best to remove this `println!`, since that will go to stdout, but if you want to keep the line in for debugging purposes, just change it to `debug!`.  Otherwise, remove completely.